docs/resource_management/RESOURCE_SCALING.md

```diff
@@ -0,0 +1,61 @@
+
+## Horizontal scaling
+The Helm charts provision one `StatefulSet` by default. In order to horizontally scale up or down the cluster `kubectl scale` can be used at runtime to provision a multi-node Data Center cluster, with no further configuration required (although note that the Ingress must support cookie-based session affinity in order for the products to work correctly in a multi-node configuration). Here is the syntax for scaling up/down the Data Center cluster:
+```
+kubectl scale statefulsets <statefulsetset-name> --replicas=n
+```
+For details on the `cpu` and `memory` requirements see [Resource requests and limits](#Resource-requests-and-limits) 
+
+## Vertical scaling
+The resource `requests` and `limits` for a `StatefulSet` can be defined either post product deployment or for deployments that are already running within the Kubernetes cluster. Take note that vertical scaling will result in the pod being re-created with the updated values
+
+### Prior to deployment
+Before performing a helm install update the appropriate products `values.yaml` `container` stanza with the desired `requests` and `limits` values i.e. 
+```yaml
+ container: 
+  limits:
+    cpu: "4"
+    memory: "4G"
+  requests:
+    cpu: "2"
+    memory: "2G"
+```
+
+### Post deployment
+For existing deployments the `requests` and `limits` values can be dynamically updated either declaratively or intrinsically 
+
+#### Declaratively
+This the preferred approach as it keeps the state of the cluster, and the helm charts themselves in sync.
+1. Update `values.yaml` appropriately
+2. Apply the patch:
+```shell
+helm upgrade <release> <chart> -f <values file>
+```
+
+#### Intrinsically
+Using `kubectl edit` on the appropriate `StatefulSet` the respective `cpu` and `memory` values can be modified. Saving the changes will then result in the existing product pod(s) being re-provisioned with the updated values.
+
+
+## Resource requests and limits
+To ensure that Kubernetes appropriately schedules resources, the respective product `values.yaml` are configured with default `cpu` and `memory` [resource request values](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/).
+
+#### Resource requests
+The default resource requests that are used for each product are defined below. Take note that these values are geared toward small data sets. For larger enterprise deployments refer to the data  enter infrastructure recommendations [here](https://confluence.atlassian.com/enterprise/data-center-infrastructure-recommendations-972333478.html) .  Using the [formula](#Memory-request-sizing), the `memory` specific values are derived from the default `JVM` requirements defined for each product's Docker container.
```

Comment:
`). Using`

Reply:
the respective product `values.yaml` is configured